### DISCUSSION TITLE
<title> Github Page failed to be searched by google?

### DISCUSSION THREAD

**Discussion by LeslieTrue** [community/community#46552] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all, 

A week ago, I set up my github page in the repo [https://github.com/LeslieTrue/website](url) but found that it doesn't show up in the result of search engine, i.e. google, after searching either site:xxxx.github.io or site:tianzhechu.com(which is may own domain linked to the page).

I also searched for some methods to fix this:
1. Wait. I've waited for 1 week but I think simply waiting doesn't work since another page of mine, estabilished several months ago, can't be searched via google either.
2. Google Console. I set up the site map and regisiter for Google Console on Feb 4th. It says that my page has been crawled and indexed but, you know, I still can't search it.

Any help or similar experience? Thank you all in advance!

**Comment by yoannchaudet** [comment]
I am not aware of a specific issue with Google indexing and GitHub Pages. Especially if you have a custom domain.

I see your site just fine on DuckDuckGo for instance.

![image](https://user-images.githubusercontent.com/14911070/217129923-94981a7c-7077-4c50-962e-6474c29a918d.png)

If you are using the Google Search Console, you are already probably doing the right thing. There are a lot of tags and things you can do to improve your SEO and eventually show up in Google search results. Google has plenty of documentation on that topic here: https://developers.google.com/search/docs/fundamentals/seo-starter-guide

**Reply by LeslieTrue** [reply]
Thank you so much! Actually it works on google, really mysterious...
